frontend/src/utils/generate-github-auth-url.ts

```diff
@@ -7,5 +7,6 @@
 export const generateGitHubAuthUrl = (clientId: string, requestUrl: URL) => {
   const redirectUri = `${requestUrl.origin}/oauth/github/callback`;
   const scope = "repo,user,workflow,offline_access";
-  return `https://github.com/login/oauth/authorize?client_id=${clientId}&redirect_uri=${encodeURIComponent(redirectUri)}&scope=${encodeURIComponent(scope)}`;
+  // return `https://github.com/login/oauth/authorize?client_id=${clientId}&redirect_uri=${encodeURIComponent(redirectUri)}&scope=${encodeURIComponent(scope)}`;
+  return `http://localhost:8080/realms/allhandsgithub/protocol/openid-connect/auth?client_id=allhandsgithub&response_type=code&redirect_uri=${encodeURIComponent(redirectUri)}&scope=openid+email+profile&state=some-state-value&nonce=222`;
```

Comment:
I think this is really the only thing we need to change here. We should probably have a `getAuthURL` on the config that returns this, which could be configured to different auth providers

The other stuff here can probably get removed (e.g. no need for the OSS to have a "keycloak token" input, and we can handle the refresh logic in the SaaS backend)